tests/queries/0_stateless/03167_base64_url_functions_sh.sh

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2155
+
+set -e
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+
+urls=(
```

Comment:
There are already some tests for base64URL in tests/queries/0_stateless/03167_base64_url_functions.sql

Consider merging these into this file (only if that makes sense) to keep things together.

Reply:
It's ok to have multiple tests on single feature)